### ISSUE TITLE
Decide On How Often A Person May Receive Travel Fund Assistance

### ISSUE THREAD

**Issue by bensternthal** [openjs-foundation/cross-project-council#1239]
The initial travel fund work proposed that a person may only receive trave funding once per year. This issue tracks discussing this limit and deciding what, if any limit the CPC wants to impose. 

_Originally posted by @mhdawson in https://github.com/openjs-foundation/cross-project-council/pull/1230#discussion_r1457905210_

**Comment by ljharb** [comment]
I'd love to hear the original rationale for that proposed cap, since this doesn't match the reality of our actual constraints nor of actual community member participation.

**Comment by tobie** [comment]
I think the intention is to avoid one individual or a few individual benefitting from fund at the expense of the broader community.

As mentioned before, I think we should just state that intention rather than come up with some possibly harmful attempt at solving it. That why I suggested a higher-level comment along the line of: _"Funding will be allocated fairly so that no single individual or group of individual benefits disproportionately from it."_

**Comment by ljharb** [comment]
Agree, and also, “benefit” here is “amount of money” - so *that* is how any limit should be expressed.

**Comment by tobie** [comment]
> also, “benefit” here is “amount of money” - so that is how any limit should be expressed.

It's not as straight-cut, though, which is why I wouldn't spell out numbers. For example, do you benefit equally from getting 5 $200 trips paid or one $1,000 trip paid? Well it entirely depends on a combination of personal context (can you afford going otherwise? Are the super impactful trips for you? etc.), and what those trips actually are.

Imagine we're left with $1,000 and there are two outstanding requests, the fourth $200 trip of person A and the first $1,000 trip of person B.

What's the right move here? It could be giving person A $200 and person B $800. It could be giving person A $100 and person B $900. It could be giving person A nothing an 1K to person B. The right answer is: "it depends."

**Comment by mcollina** [comment]
How about we add a `-5` in our scoring rubric for every trip a person has taken in the calendar year? As a result, it's _unlikely_ that somebody would be funded twice, but it would allow us an escape hatch.

**Comment by mhdawson** [comment]
Not saying that I know the right answer to this, but setting an expectation up front is good.  If the reality is that somebody is likely to only be funded once except in exceptional cases, it is good to have that understood up front.  Adding to the rubric does not help people reading the policy understand that.  For example lets not have it be a surprise if a collaborator can only go 1 one of the 2 Node.js collaborator summits each year.   In my mind the reality is that its being driven by the limited $ available from the Foundation and the suggestion to limit $ per person per year seems to align to that better than once per year. 

Our goal is to maximize the benefit to the overall ecosystem while balancing that with being fair to all collaborators. Providing more info to collaborators that helps them make their travel fund requests balance those two is good. For example in my mind if 5 $200 travel requests will get a collaborator to 5 events instead of 1 more costly one, then that is likely better for the overall ecosystem while balancing the need to be fair to call collaborators. 

Summing up I'd prefer that there is something in the doc that sets expectations based on what is likely to happen. We can always soften the wording to something like: "The maximum a person can received from the travel fund per year is $1500, except when the CPC believes it is in the overall interest of the community to approve more than that".

**Comment by ovflowd** [comment]
> Adding to the rubric does not help people reading the policy understand that.

But wouldn't that apply to all score pieces of the scoring system? None guarantees your participation or approval of your fund request. I believe that having a -5 (or any other value) scoring based on previous fund requests in the same financial year and being upfront (communicating well that requesting more than one fund per year might as well cause the request to be rejected) to be more than enough to help people understand the policy.

It is my understanding that the whole scoring system will be documented so that anyone interested in applying for the travel fund understands what might affect the % of their request being approved or not. After all, no one applying for the travel fund should have any expectation of it being approved and should rely on that fund before it gets approved, right?

**Comment by tobie** [comment]
> setting an expectation up front is good. If the reality is that somebody is likely to only be funded once except in exceptional cases, it is good to have that understood up front.

That’s a great point. Could we make this informative rather than policy? For example by providing some estimation based on last year? For example: “Last year the fund was X, it was used to cover 23 trips for 18 contributors ranging from $200 to $1789, with an average of $815”

**Comment by tobie** [comment]
I ran some quick number on the publicly available info for 2023. Here's what this looks like: “In [2023](https://github.com/openjs-foundation/community-fund/blob/main/programs/travel-fund/2023.md), 26 funding requests were made by 22 contributors for a total requested sum of USD 47,500 with a maximum of 2 requests per contributor. All requests were granted. Requested sums per trip ranged from $750 to $5,750, with a median of $1,525 and an average of $1,825. Requested sums per individuals ranged from $1,000 to $7,350, with a median of $1,950 and an average of $2,150."

@bensternthal does this match your internal data sufficiently that we could add this to the policy?

**Comment by bensternthal** [comment]
@tobie This is close to the numbers I pulled. One notable difference is that the average actual amount paid is ~1500. It is not surprising that folks ask for more than is actually expensed.

**Comment by tobie** [comment]
Thanks. That’s good enough for this year imho. As @mcollina mentioned elsewhere, it’s worth making sure people are aware that filing expenses quickly frees up additional funds for others.

**Comment by mcollina** [comment]
IMHO I would put an hard 30 days window for submitting the expense report. Would that help @bensternthal ?

**Comment by tobie** [comment]
I think "send your expenses in quickly so we can use the funds for other people" is a much stronger incentive than "here's an arbitrary deadline."

**Comment by ljharb** [comment]
Also i think the negative consequences of “the overage from expenses isn’t available to others” is far less serious than “on day 31, you’re fully liable for the full amount”, given that the overage will likely be very small but the entire amount hugely punitive for some.
